src/utils/tflHelpers.ts

```diff
@@ -0,0 +1,21 @@
+import TflFeedData from '../interfaces/queryResults/tflFeedData';
+
+export function processTFLFeedData(data: TflFeedData): TflFeedData {
+  return Object.assign({}, ...Object.keys(data).map((key) => ({ [key]: escapeString(data[key]) }))) as TflFeedData;
+}
+
+export function escapeString(str: any): string {
```

Comment:
Are we not escaping strings using backslashes?

Reply:
Nevermind, just read the spec

Reply:
Yeah I promise it's not me just being really weird